pkg/ingress/apisix/tls.go

```diff
@@ -32,8 +31,6 @@ type ApisixTLSCRD configv1.ApisixTls
 func (as *ApisixTLSCRD) Convert(sc Secreter) (*apisix.Ssl, error) {
 	name := as.Name
 	namespace := as.Namespace
-	_, group := BuildAnnotation(as.Annotations)
-	conf.AddGroup(group)
```

Comment:
We also need to explain on the document that the use of ingress.class has changed and is consistent with the official k8s

Reply:
Will add to document in the `ApisixRoute v2alpha1 ` PR .